### PR TITLE
percentage based start seed data + seeded tr/va/trva/te split + bugfix

### LINKED ISSUE #81: Parametrize  the amount of starting seed training data
Now the start seed training data is just the first song. 
What we want is to implement functionality to pick for example X amount of songs or a x% subset of the training data as starting seed training data for AL.

This should be fixed in Viability phase . py and user phase .py(Or whatever the user phase files name is).

### LINKED ISSUE #83: fix get_mse_valence to return valence
fix get_mse_valence to return valence (it now returns arousal

### LINKED ISSUE #84: enhance feature selection to use train, val, test split (seeded)
enhance feature selection to use train, val, test split (seeded)

### PR DESCRIPTION
**Pull request by tediiieee**
This pull request includes several small fixes/features:
- Middle two commits (feature_selection) relates to #84, seeded ordinary split with the addition of a validation set!(defaults to tr/va/te 60/20/20 split) However it also saves a trva data that of course is both the tr and va data. This can(should) be used for training in the evaluation phase)
- 
- Fixed bug where the wrong values where return (arousal instead of valence)
- Added support/fixed such that the starting seed training data now is percentage based defaulted to 10%.

Closes #84 
Closes #83 
Closes #81

### CODE REVIEW

**Review by Ludvig-Joborn — CHANGES_REQUESTED** [review]
See comments.

File: src/viability_phase.py [hunk 1]
```diff
@@ -161,3 +173,92 @@ def _evaluate(lp: LearningProfile, num_iterations: int):
         MSE_valence_arr.append(MSE_valence)
 
     return MSE_arousal_arr, MSE_valence_arr
+
```
[the remark this answers is not in the document]
Comment (tediiieee) [reply src/viability_phase.py]: FixD

File: src/viability_phase.py [hunk 2]
```diff
@@ -9,7 +9,8 @@
 import numpy as np
 
 
-def viability_phase(learning_profiles: list, num_iterations: int = -1):
+def viability_phase(learning_profiles: list, num_iterations: int = 1,
```
[the remark this answers is not in the document]
Comment (tediiieee) [reply src/viability_phase.py]: FixD

**Review by Ludvig-Joborn — APPROVED** [review]
Great job!
